fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,51 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+DoSliceTimeCorrection=`opts_GetOpt1 "--doslicetime" $@`                 # Whether to do slicetime correction (TRUE), FALSE to omit
+SliceTimerCorrectionParameters=`opts_GetOpt1 "--slicetimerparams" $@`   # A '@' separated list of FSL slicetimer options. Please see FSL slicetimer documentation for details.
+                                                                        # Verbose (-v) is already turned on. TR (-r) is specified using --tr parameter.
```

Comment:
Perhaps provide a simple example as an illustration:
e.g., 
`--slicetimerparams=--odd@--ocustom=<CustomInterleaveFile>`

Reply:
Since this option could contain a filename, and filenames could have whitespace in them, please change the code to:
```
SliceTimerCorrectionParameters=$(opts_GetOpt1 "--slicetimerparams" "$@")
```
Ideally, all instances of `$@` should be quoted as `"$@"`, and `$()` is the more modern style of command substitution that allows better parsing and more obvious reading of the line.  However, you can leave old lines of code as-is, we aren't expecting you to clean up our messes.

Reply:
Also, if there is a default for this option, it may need special care, as the old option parsing code used a bad hack to set defaults, which prevent proper quoting.

Reply:
Thanks, I changed it. No defaults for this one. Onus on the user to set it correctly.